core/config.mk

```diff
@@ -137,7 +137,9 @@ COMMON_RELEASE_CPPFLAGS:= $(COMMON_RELEASE_CFLAGS)
 
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit